conf/documentation.conf

```diff
@@ -465,6 +472,13 @@ Location of the iptables binary. Only necessary to change if you are
 not running the RPMed version.
 EOT
 
+[services.ip6tables_binary]
+type=hidden
+description=<<EOT
+Location of the ip6tables binary. Only necessary to change if you are
+not running the RPMed version.
```

Comment:
"RPMed version" - how about the Debian packages? Should we simply say "if you are not running the official packaged version" ?